process/track/blockProcessor.go

```diff
@@ -25,7 +25,8 @@ type blockProcessor struct {
 	finalMetachainHeadersNotifier         blockNotifierHandler
 	roundHandler                          process.RoundHandler
 
-	blockFinality uint64
+	blockFinality                   uint64
+	shouldProcessReceivedHeaderFunc func(_ data.HeaderHandler) bool
```

Comment:
Also here, could have kept the original param names

Reply:
Removed, as it is used only from outside of the component and is not necessary to be overwritten

Reply:
Done